enterprise/frontend/src/metabase-enterprise/embedding/components/EmbeddingAppSameSiteCookieDescription/EmbeddingAppSameSiteCookieDescription.tsx

```diff
@@ -45,7 +45,7 @@ function AuthorizedOriginsNote() {
       <SameSiteAlert variant="warning" hasBorder>
         <Center>
           <Text>{jt`You should probably change this setting to ${(
-            <Text span fw="bold">
+            <Text key="inner" span fw="bold">
```

Comment:
just a random key to make react shut up
<img width="1013" alt="image" src="https://github.com/metabase/metabase/assets/1914270/14054c6f-4648-4719-bb9b-dd98151892b6">